openshift-integration.md

```diff
@@ -0,0 +1,105 @@
+# The OpenShift integration user guide
```

Comment:
Can you add that page to the `summary.md` page, so that it is added to the menu?